### PR TITLE
Update Engine & Interactive-UX Working Group Member lists

### PR DESCRIPTION
**Pull request by kilasuit**
# PR Summary

Adds Friedrich Weinmann @FriedrichWeinmann & Steven Bucher @StevenBucher98 to Interactive UX WG list
Adds  Staffan Gustafsson @powercode to Engine WG

## PR Context

Ensures doc containing the WG Member list is up to date

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

File: docs/community/working-group-definitions.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ These topics include (but are not limited to):
 * @SeeminglyScience
 * @sdwheeler
 * @kilasuit
+* @FriedrichWeinmann
```
Comment (StevenBucher98) [inline docs/community/working-group-definitions.md:94]:
```suggestion
* @FriedrichWeinmann
* @StevenBucher98 
```

Comment (StevenBucher98) [reply docs/community/working-group-definitions.md:94]: I'm not included yet in the list either 😄

Comment (kilasuit) [reply docs/community/working-group-definitions.md:94]: Thanks for the catch - updated!

**Review by StevenBucher98 — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@kilasuit Please resolve the failure in markdown-line CI.

**Comment by kilasuit** [comment]
@daxian-dbw I was looking into this & the tools used for linting as I disagree with the error raised. In my opinion this is something we should ignore being an issue as both `bug fixes`& `bugfixes`are acceptable terminology until I/others can work out how to silence this error which I couldn't work out exactly where to do this in looking yesterday due to mix of tooling & their differing configuration settings across them all being used here. But it's on my list of things to dig into further at some point in the not too distant future.

**Comment by kilasuit** [comment]
Also that error shouldn't really be an error but a warning as its a language preference thing vs it being something more like `PowerShell`vs `powershell`& is yet another reason why I disagree with it & think we should ignore it for now.

**Comment by kilasuit** [comment]
@daxian-dbw I could make the additional change to resolve the failure however I think that it's outside the scope of this PR so I don't think should be added especially for the reasons that I listed above, as such can we please merge this & I'll try and find where to update the tooling that throws the error to make it a warning instead which would be better suited for us going forward.

**Comment by iSazonov** [comment]
Will #20996 fix these markdown issues?

**Comment by kilasuit** [comment]
> Will #20996 fix these markdown issues?

That'll only fix any checks for links, this particular linting issue is caused by this [line in textlint-rule-terminology](https://github.com/sapegin/textlint-rule-terminology/blob/78b8deebbd37a2ae2adc9127614d537f1c1e5faa/terms.jsonc#L109) which I raised [this issue](https://github.com/sapegin/textlint-rule-terminology/issues/73) & hope the maintainer for that project will resolve this.

**Comment by kilasuit** [comment]
I had attempted some changes as had been recommended in https://github.com/super-linter/super-linter/issues/5101 however as they were still making the `markdown-lint` workflow fail I've reverted those changed. It may instead be better to replace super-linter with https://github.com/marketplace/actions/markdownlint-cli2-action (as we already make use of the vscode extension `DavidAnson.vscode-markdownlint` ) especially if we are only linting Markdown files. I feel that sort of change is out of scope for this PR though

**Comment by kilasuit** [comment]
I had some additional feedback as possible changes to the workflow file & this then lead to noticing some additional items that could get resolved,
I don't think that I can get back to the latest build error until next week but if I can then I will do so

**Comment by StevenBucher98** [comment]
@kilasuit I created and merged a PR for updating all the additional WG memebers, it does NOT include the members you have here. I suggest rebasing as I fixed most of the linting issues in this PR: https://github.com/PowerShell/PowerShell/pull/21091

**Comment by kilasuit** [comment]
@StevenBucher98 / @daxian-dbw this is now ready for merging. Sorrry it took so long.

**Pull request merged**